app/presenters/organisations/documents_presenter.rb

```diff
@@ -174,5 +204,14 @@ def latest_statistics
       @latest_statistics ||= search_rummager(filter_government_document_supertype: "statistics")
       search_results_to_documents(@latest_statistics)
     end
+
+    def promotions_child_column_class(number_of_items)
+      return "column-half" if number_of_items == 2
+      "column-one-third" if number_of_items == 3
+    end
+
+    def promotional_feature_link(link)
+      link if link.length.positive?
```

Comment:
This will error if `item["href"]` is `nil` - not sure if that's ever a possibility but good to handle just in case. You can do `link if link && link.length.positive?`